api/scpca_portal/test/management/commands/test_create_portal_metadata.py

```diff
@@ -109,14 +78,9 @@ def test_create_portal_metadata(self, mock_upload_output_file):
         computed_file = computed_files.first()
         # Make sure the computed file size is as expected range
         self.assertEqualWithVariance(computed_file.size_in_bytes, 8430)
-        # Make sure all fields match the download configuration values
-        download_config = {
-            "modality": None,
-            "format": None,
-            "includes_merged": False,
-            "metadata_only": True,
-            "portal_metadata_only": True,
-        }
+        # Make sure all fields match the download configuration (excludes excludes_multiplexed)
+        download_config = common.PORTAL_METADATA_DOWNLOAD_CONFIG.copy()
+        download_config.pop("excludes_multiplexed", None)
```

Comment:
lets replace this with assertions for expected values in the same way we handle `file_size_in_bytes`

Reply:
I've added the individual assertions for the fields defined in the portal metadata `computed_file` in 938cd6a0175d8ef605161a918572bed7cad78380. This PR is ready for another look. Thank you David!